source/docs/advanced/payments/adyen.md

```diff
@@ -189,6 +210,100 @@ Add the following line to your `web/theme/modules/_modules.scss` file to load th
 @import "~theme/modules/Adyen/dropinCustomizations";
 ```
 
+### Add webhook
+
+From your adyen Customer Area under `Developers > Webhooks` click on `+ Webkook` button on the right top corner then click on `Add` for the `Standard Notification` type. Fill the fields as follows:
```

Comment:
```suggestion
In your Adyen Customer Area under `Developers > Webhooks` click on the `+ Webkook` button on the right top corner then click on `Add` for the `Standard Notification` type. Fill the fields as follows:
```

Reply:
done in 9c2b629bbcb77cc2c8ca238288da9815de60076a